frontend/src/constants/zipCodes.ts

```diff
@@ -1,76 +1,11 @@
-import { chain, keys, mapValues } from 'lodash'
+import { chain, flow, keys, mapValues, sortBy } from 'lodash'
 import { serializeServiceArea } from '../utils/serializers'
 import { State } from './states'
+import { ZIPS_BY_COUNTY_BY_STATE } from './zipsByCountyByStateCode'
```

Comment:
Typo, should be zipCodesByCountyByState

Reply:
Haha well that explains the build failure..